### PR TITLE
Add missing things to release notes [ci skip]

### PR DESCRIPTION
**Pull request by prathamesh-sonpatki**
- https://github.com/rails/rails/pull/18721
- https://github.com/rails/rails/pull/11898
- https://github.com/rails/rails/pull/18194
- https://github.com/rails/rails/pull/18225
- https://github.com/rails/rails/pull/18244
- https://github.com/rails/rails/commit/edc2b7718725016e988089b5fb6d6fb9d6e16882
- https://github.com/rails/rails/pull/19686

r? @rafaelfranca

### CODE REVIEW

File: guides/source/5_0_release_notes.md [hunk 1]
```diff
@@ -671,10 +674,24 @@ Please refer to the [Changelog][active-record] for detailed changes.
...
+
+*   Added ability to call `ActionRecord::Relation#update` on relation objects
+    which will run validations on callbacks on all objects in the relation.
+    ([Pull Request](https://github.com/rails/rails/pull/11898))
+
+*   Added touch option to the `save` method so that records can be saved without
+    updating timestamps.
+    ([Pull Request](https://github.com/rails/rails/pull/18225))
+
+*   Added expression indexes and pperator classes support for PostgreSQL.
```
Comment (vipulnsward) [inline guides/source/5_0_release_notes.md]: operator

File: guides/source/5_0_release_notes.md [hunk 2]
```diff
@@ -671,10 +674,24 @@ Please refer to the [Changelog][active-record] for detailed changes.
...
+    Previously this was only supported on the deprecated `mysql` legacy adapter.
     To enable, set `prepared_statements: true` in config/database.yml.
-    ([Pull Request](https://github.com/rails/rails/pull/23461))        
+    ([Pull Request](https://github.com/rails/rails/pull/23461))
+
+*   Added ability to call `ActionRecord::Relation#update` on relation objects
+    which will run validations on callbacks on all objects in the relation.
+    ([Pull Request](https://github.com/rails/rails/pull/11898))
+
+*   Added touch option to the `save` method so that records can be saved without
```
Comment (vipulnsward) [inline guides/source/5_0_release_notes.md]: `touch`

File: guides/source/5_0_release_notes.md [hunk 3]
```diff
@@ -671,10 +674,24 @@ Please refer to the [Changelog][active-record] for detailed changes.
...
+    updating timestamps.
+    ([Pull Request](https://github.com/rails/rails/pull/18225))
+
+*   Added expression indexes and pperator classes support for PostgreSQL.
+    ([commit](https://github.com/rails/rails/commit/edc2b7718725016e988089b5fb6d6fb9d6e16882))
+
+*   Added option to index errors of nested attributes.
```
Comment (vipulnsward) [inline guides/source/5_0_release_notes.md]: Added `:index_errors` option..

File: guides/source/5_0_release_notes.md [hunk 4]
```diff
@@ -691,6 +708,9 @@ Please refer to the [Changelog][active-model] for detailed changes.
     [activemodel-serializers-xml](https://github.com/rails/activemodel-serializers-xml) gem.
     ([Pull Request](https://github.com/rails/rails/pull/21161))
 
+*   Removed `ActionController::ModelNaming` module.
+    ([Pull Request](https://github.com/rails/rails/pull/18194))
```
Comment (vipulnsward) [inline guides/source/5_0_release_notes.md]: Don't think it should be part of release notes, its din't have a deprecation or changelog entry

Comment (prathamesh-sonpatki) [inline guides/source/5_0_release_notes.md]: Added CHANGELOG here https://github.com/rails/rails/pull/25322. There is no deprecation for it as it was unused.

File: guides/source/5_0_release_notes.md (hunk 2 above)
Comment (prathamesh-sonpatki) [inline guides/source/5_0_release_notes.md]: Fixed.

File: guides/source/5_0_release_notes.md (hunk 3 above)
Comment (prathamesh-sonpatki) [inline guides/source/5_0_release_notes.md]: Fixed

**Comment by maclover7** [comment]
Backported to `5-0-stable` via 45980d86e435887ef80a70c6c20a4b4fffc3a453, `5-0-0` via 45980d86e435887ef80a70c6c20a4b4fffc3a453t (same commit).

**Pull request merged**